### DISCUSSION TITLE
Multiple Webhook URL on GitHub App

### DISCUSSION THREAD

**Discussion by taisiiakholodova** [community/community#53779] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

HI GitHub Community

I am working on GitHub App which is used by multiple environments(dev/staging/prod), so  all the environments should receive webhook notification.

Is there a way to set multiple Webhook URL on GitHub App?

P.S. feel free to suggest alternative solution for the problem. I have two options for now:
1) Create an app per each environment
2) Create a proxy on webhook side to re-sent data to all environments

Thank you!

**Comment by bselwe** [comment]
I'd suggest creating a separate GitHub App per each environment. This way, you can easily modify App permissions, events, callback URLs when in development or testing on staging, without affecting the production app.
